# This msg file specifies the 3D pose which includes
# (1) position (x,y,z) (2) orientation (in the format of quaternion x,y,z,w)

geometry_msgs/Point position
geometry_msgs/Quaternion orientation
